apps/emqx_auth/src/emqx_authz/emqx_authz_schema.erl

```diff
@@ -136,7 +136,7 @@ authz_fields() ->
     [
         {sources,
             ?HOCON(
-                ?ARRAY(?UNION(UnionMemberSelector)),
+                ?ARRAY(hoconsc:union(UnionMemberSelector)),
```

Comment:
```suggestion
                hoconsc:array(hoconsc:union(UnionMemberSelector)),
```
?

Reply:
thanks, will fix in a follow-up or if this pr needs another fix in the ci.